package.json

```diff
@@ -107,7 +107,9 @@
       "!src/**/index.ts",
       "!src/reportWebVitals.ts",
       "!src/serviceWorker.ts",
-      "!src/components/TaskPage/*.{ts,tsx}"
+      "!src/components/TaskPage/*.{ts,tsx}",
+      "!src/components/EmptyStateSVG/*.{ts,tsx}",
+      "!src/components/LogoSVG/*.{ts,tsx}"
```

Comment:
I'm curious why these files specifically were added to this part?

Reply:
oops 🙊 I removed the ones for the SVGs and added tests for them.